src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.println(ball);
+        }
```

Comment:
According to the checklist, magic numbers should be replaced with named constants. The number '3' here is a magic number and should be declared as a constant at the beginning of the class, with a clear name indicating its purpose.